_layouts/base.html

```diff
@@ -10,6 +10,41 @@
         {% if page.description %}<meta name="description" content="{{ page.description }}">{% endif %}
         <link href="{{ site.baseurl }}/css/custom.css" rel="stylesheet" type="text/css">
         <link rel="stylesheet" href="https://cdn.rawgit.com/jpswalsh/academicons/master/css/academicons.min.css">
+        <!--Below is the schema annotation for blogposts-->
+        {% if page.layout == 'post' %}
+        <script type="application/ld+json">
+          {
+            "@context": "https://schema.org",
+            "@type": "BlogPosting",
+            "mainEntityOfPage": {
+              "@type": "WebPage",
+              "@id": "{{ site.url }}/posts/"
+              },
+            "headline": "{{ page.title }}",
+            "description": "{{ page.excerpt | strip_html | strip_newlines | truncate: 200 }}",
+            "image": "{{ page.image }}",
+            {% for author in page.authors %}
+            "author": {
+                  "@type": "Person",
+                  "name": "{{ author }}",
+                  "url": 
+                  "{{ site.url }}/people#{{ author }}",
+                },
+              {% endfor %}
+            "publisher": {
+              "@type": "Organization",
+              "name": "Open Life Science",
+                "logo": {
+                  "@type": "ImageObject",
+                  "url": "{{ site.url }}/images/logo.png"
+                }
+              },
+            "datePublished": "
+              {{ page.date | date: "%Y-%m-%d" }}",
+            "dateModified": ""
```

Comment:
If empty, better to remove

Reply:
I was searching for ways to display the `dateModified` but it would require a plugin that is not supported by githubpages (again). So, removing now.